src/app/features/side-bar/side-bar.component.scss

```diff
@@ -6,7 +6,10 @@
 mat-nav-list {
     padding: 0;
 
-    .sidebar-link {
-        @include mat.typography-level($custom-typography, 'headline-1'); //not working but the <a> tag in header is working
+}
+
+.sidebar-link {
+    span {
+        font-size: 16px;
```

Comment:
Add font-size to _variable.scss
For ex: $font-size-xs, $font-size-s, $font-size-m, $font-size-l etc